src/main/scala/symsim/examples/concrete/cliffwalking/CliffWalking.scala

```diff
@@ -0,0 +1,115 @@
+package symsim
+package examples.concrete.cliffwalking
```

Comment:
the name of the package (and of the corresponding directory) should be `cliffWalking`